index.html

```diff
@@ -6,6 +6,24 @@
     <title>Les 3 - Opdracht CSS Grid</title>
 </head>
 <body>
-
+<div class="container">
+    <header class="items" id="item-a">Header</header>
+    <header class="items" id="item-c">C</header>
+    <main class="items" id="item-b">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ab accusamus beatae
+        corporis culpa ea eveniet fuga ipsam iste molestiae molestias necessitatibus nisi nulla praesentium, sapiente
+        sint suscipit tenetur totam. A ab aliquid architecto corporis cumque delectus distinctio eligendi eum ex
+        exercitationem fugiat harum impedit in ipsa labore magni officia praesentium provident, quidem quod repellendus
+        repudiandae sapiente sunt vel voluptatum. Dignissimos, ex, suscipit? Aliquid animi dolorem expedita, harum
+        impedit ipsam laborum molestiae nemo nobis non nulla pariatur perferendis quam sed ullam. Ab adipisci aliquid
+        animi blanditiis consectetur consequatur, consequuntur dignissimos error ex excepturi fuga illo inventore iste
+        labore laborum magni minus necessitatibus nulla quae quas quo reiciendis rem reprehenderit tempore vel veniam,
+        vitae. Aliquid asperiores cupiditate deserunt error et facilis fuga ipsum, itaque minus molestiae omnis quaerat
+        quisquam recusandae rem sint tempora ullam! A consequuntur debitis dolores ducimus exercitationem fugiat, illum
+        in iusto labore necessitatibus sit, tempore tenetur unde vel veritatis.
+    </main>
+    <main class="items" id="item-e">E</main>
```

Comment:
Ziet er keurig uit, ik weet alleen niet of het gebruikelijk is om meerdere mains en headers te gebruiken, misschien even checken?

Reply:
Ja dat wist ik zelf ook niet, maar vond het ook geen duidelijke header ofzo die over de hele breedte ging. Maar dit ga ik inderdaad even checken :)